sequence.go

```diff
@@ -0,0 +1,84 @@
+package gween
+
+type (
+	// Sequence represents a sequence of Tweens, executed one after the other.
+	Sequence struct {
+		Tweens []*Tween
+		index  int
+	}
+)
+
+// NewSequence returns a new Sequence object.
+func NewSequence() *Sequence {
```

Comment:
You could probably just instantiate the sequence with tweens 

```golang
func NewSequence(tweens ...*Tween) *Sequence {
	seq := &Sequence{
		Tweens: tweens,
	}
	return seq
}
```
no need to worry about it being nil because `append` and `range` handle a nil slice just fine in the other methods